vyper/exceptions.py

```diff
@@ -128,12 +129,15 @@ def format_annotation(self, value):
 
             # TODO: handle cases where module is None or vy_ast.Module
             if module_node.get("path") not in (None, "<unknown>"):
-                node_msg = f'{node_msg}contract "{module_node.path}:{node.lineno}", '
+                node_msg = self._format_contract_details(node_msg, module_node.path, node.lineno)
```

Comment:
here we use `module_node.path`, but in the exception handlers added in this commit, `resolved_path` is used